docs/source/reference/index.rst

```diff
@@ -136,4 +136,13 @@ TensorFlow
    :toctree: generated/
    :nosignatures:
 
-   optuna_integration.TFKerasPruningCallback
+   optuna.integration.TFKerasPruningCallback
+
+XGBoost
+----------
```

Comment:
```suggestion
XGBoost
-------
```

Reply:
... and this minor correction has been done as well...